src/bundle/Resources/public/scss/_tag-view-select.scss

```diff
@@ -0,0 +1,63 @@
+.ibexa-tag-view-select {
+    border: calculateRem(1px) solid $ibexa-color-dark-200;
+    border-radius: $ibexa-border-radius;
+    padding: calculateRem(24px);
+    letter-spacing: calculateRem(0.12px);
+
+    &__info {
+        @include ibexa-font-sans-serif();
+
+        font-size: $ibexa-text-font-size-medium;
+        color: $ibexa-black;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+
+    &__selected_list {
+        display: flex;
+        margin: calculateRem(10px) 0;
+    }
+
+    &__selected_item {
+        display: inline-flex;
+        align-items: center;
+        height:calculateRem(24px);
+        padding: 0 calculateRem(8px);
+        border-radius: $ibexa-border-radius;
+        background-color: $ibexa-color-light-400;
+        font-size: $ibexa-text-font-size-small;
+        line-height: calculateRem(18px);
+
+        & + & {
```

Comment:
Can it be replaced with `:not(:first-of-type)`? I think it would be easier to understand what it does.